conda/faiss/meta.yaml

```diff
@@ -40,7 +40,7 @@ outputs:
       host:
         - mkl =2018
       run:
-        - mkl >=2018
+        - mkl >=2018,<2021
```

Comment:
`mkl-2021.2.0-haa95532_296` doesn't contain `mkl_rt.dll` (it contains `mkl_rt.1.dll` instead). We need to constrain `mkl` as `<2021` if `faiss` should be built with `mkl=2018` (linked to `mkl_rt.dll`) .